hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el

```diff
@@ -0,0 +1,92 @@
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "2423918326 3522900596"
+INSTANCE: tb.dut.gen_partitions[4].gen_buffered.u_part_buf
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "2423918326 3522900596"
+INSTANCE: tb.dut.gen_partitions[5].gen_buffered.u_part_buf
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "2423918326 3522900596"
+INSTANCE: tb.dut.gen_partitions[6].gen_lifecycle.u_part_buf
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "1031540710 109269272"
+INSTANCE: tb.dut.gen_partitions[0].gen_unbuffered.u_part_unbuf
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->AccessError "6->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->AccessError "7->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: unreachable for unbuffered partition only"
+Transition AccessError->CheckFailError "5->6"
+CHECKSUM: "1031540710 109269272"
+INSTANCE: tb.dut.gen_partitions[1].gen_unbuffered.u_part_unbuf
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->AccessError "6->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->AccessError "7->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: unreachable for unbuffered partition only"
+Transition AccessError->CheckFailError "5->6"
+CHECKSUM: "4056579122 2961818352"
+INSTANCE: tb.dut.u_otp_ctrl_dai
+Fsm error_q "2961818352"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->AccessError "7->5"
```

Comment:
these UNR should be generated from the VCS UNR flow, right? 
since we should re-visit this again, can we add a TODO

Reply:
Sounds good. I will add a TODO for that. Yes it should generate from the UNR.